samples/wifi/radio_test/src/nrf_wifi_radio_test_shell.c

```diff
@@ -839,31 +840,39 @@ void nrf_wifi_radio_test_get_max_tx_power_params(void)
 	 *So,multiply 4 to MAX_TX_PWR_RADIO_TEST and
 	 *configure the RF params corresponding to Max TX power
 	 */
-	ctx->conf_params.rf_params[NRF_WIFI_RF_PARAMS_OFF_CALIB_PWR2G] =
-	(MAX_TX_PWR_RADIO_TEST << 2);
+	ctx->conf_params.rf_params[NRF_WIFI_TX_PWR_CEIL_BYTE_OFFSET +
```

Comment:
(Generic comment) Can't we now just cast this to struct nrf_wifi_phy_rf_params and assign to the member elements of the structure ?